include/openthread/error.h

```diff
@@ -223,6 +223,11 @@ typedef enum otError
      */
     OT_ERROR_LINK_MARGIN_LOW = 34,
 
+    /**
+     * Input (CLI) command is invalid.
+     */
+    OT_ERROR_INVALID_COMMAND = 35,
```

Comment:
I think there are many other command parsing code that we should change from returning `OT_ERROR_PARSE` to returning `OT_ERROR_INVALID_COMMAND`. Still `work-in-progress` ?

Reply:
Thanks for mention, I think sub-commands like `udp send` should also return `OT_ERROR_COMMAND`. Will do.